src/main/java/cloud/fogbow/ras/constants/Messages.java

```diff
@@ -4,23 +4,25 @@
 
     public static class Exception {
         public static final String ACTIVE_SOURCE_COMPUTE = "The source compute needs to be paused or hibernated before taking the snapshot.";
-        public static final String ADMIN_ROLE_DOES_NOT_EXIST = "Admin role '%s' does not exist.";
+        public static final String ADMIN_ROLE_NOT_SPECIFIED = "Admin role is not specified in the configuration file.";
```

Comment:
Should it not use the Exception that was defined in Common?

Reply:
These are two different exception cases. ADMIN_ROLE_NOT_SPECIFIED is used when the configuration does not contain a property specifying which role is the admin role. ADMIN_ROLE_DOES_NOT_EXIST is used when the policy does not describe the admin role specified by the configuration property.